tests/unit/test_local_sparql_ds.py

```diff
@@ -32,6 +48,16 @@ def test_load_local_query_from_file_sparql_fetch_tabular():
     local_rdf_ds.fetch_tabular()
 
 
+def test_load_local_query_from_file_substitution():
```

Comment:
well done, adding tests; 
I am not sure that the simplest case is covered: a given SPARQL query, not from file, without any substitution.

Just noticed an assert is missing there `test_load_local_sparql_fetch_tabular()`

Reply:
in fact, more asserts are missing; can you please look into that? 
* test_load_local_query_from_file_sparql_fetch_tabular
* test_load_local_sparql_fetch_tabular
* test_load_local_sparql_fetch_tree
* test_load_local_query_from_file_and_direct_text_sparql_fetch_tabular
*